README.md

```diff
@@ -1,18 +1,36 @@
 # Mautic custom email settings plugin
-This feature allows Mautic to send different emails through different mail API keys.
+This feature allows Mautic to send different emails through different mail API keys and different email services. Currently plugin support only Sparkpost and Sendgrid API services.
```

Comment:
This plugin allows Mautic to switch between different email services - Sparkpost and Sendgrid. You can also add different API keys for the different email providers.